plugins/org.jkiss.dbeaver.model.event/src/org/jkiss/dbeaver/model/websocket/event/session/WSLogSenderEvent.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.model.websocket.event.session;
+
+import org.jkiss.dbeaver.model.websocket.event.WSEventType;
+
+import java.util.List;
+
+public class WSLogSenderEvent extends WSAbstractSessionEvent {
+    String asyncTaskId;
+    List<String> messages;
+    long eventTimestamp;
```

Comment:
Why are these package-private? Wouldn't adding getters, as in any other event class, be better?

I'd also like them to be marked as "final" (assuming they aren't reassigned elsewhere).